---
trajectory_msgs/MultiDOFJointTrajectory trajectory